<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountPage</class>
 <widget class="QWidget" name="AccountPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>981</width>
     <height>601</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="Cart">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <attribute name="title">
     <string>Cart</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>971</width>
       <height>571</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>969</width>
        <height>569</height>
       </rect>
      </property>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>969</width>
         <height>109</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>969</width>
         <height>109</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>220</y>
         <width>969</width>
         <height>109</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>330</y>
         <width>969</width>
         <height>109</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>440</y>
         <width>969</width>
         <height>109</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>550</y>
         <width>969</width>
         <height>109</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="History">
    <attribute name="title">
     <string>History</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="Track">
    <attribute name="title">
     <string>Track Order</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="Logout">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QLabel" name="CustomerName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>721</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Md Sakil Khan</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QLabel" name="CustomerName_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>841</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Balance: $100.00 Chase Bank *1234</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QLabel" name="CustomerIcon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>userIcon.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="CustomerName_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>841</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>123-12 123st, Qwerty, NY 12345</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
